web-server/src/components/Service/SystemLog/PlainLog.tsx

```diff
@@ -0,0 +1,9 @@
+import { Line } from '@/components/Text';
+
+export const PlainLog = ({ log }: { log: string; index: number }) => {
+  return (
+    <Line medium marginBottom={'8px'} fontFamily={'monospace'}>
```

Comment:
Try this `<Line mono marginBottom={1}>`, it should ideally give you the same effect.

Reply:
done